test/e2e-tests.sh

```diff
@@ -65,7 +65,8 @@ must_fail  "logs of taskrun before installing crd" tkn taskrun logs foo
 install_pipeline_crd
 
 # listing objects after CRD is created should not fail
-run_test  "list pipeelinerun for pipeline foo" tkn taskrun list foo
+kubectl config set-context $(kubectl config current-context) --namespace=default
```

Comment:
can you test this on openshift CI ?

Reply:
yes, tested https://github.com/openshift/tektoncd-cli/pull/35

Reply:
thanks